# Command to set actuator speed
# Component names: "vacuum", "main_brush", "side_brush", "water_pump"
string component
uint8 speed  # 0-100%
